src/Microsoft.Health.Fhir.Core/Exceptions/RetriableTaskException.cs

```diff
@@ -0,0 +1,21 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using Microsoft.Health.Fhir.Core.Models;
+
+namespace Microsoft.Health.Fhir.Core.Exceptions
+{
+    public class RetriableTaskException : FhirException
+    {
+        public RetriableTaskException(string message)
+            : base(message)
+        {
+            Issues.Add(new OperationOutcomeIssue(
```

Comment:
Are we planning on returning this information back to the user? I don't think these exceptions need to be of type `FhirException` unless we plan to return some of this information back to user.

Reply:
We do not expect customer see this exception, move to task management project.